web-marketplace/src/lib/queries/react-query/registry-server/graphql/getProjectByIdQuery/getProjectByIdQuery.ts

```diff
@@ -29,19 +29,9 @@ export const getProjectByIdQuery = ({
       });
 
       if (projectById.data?.projectById?.metadata) {
-        projectById.data.projectById.metadata = await jsonLdCompact({
-          // NOTE: We set the context here to prevent a prefix confusion
-          // error when no context is provided and field values with the
-          // prefixes are present within the project metadata.
-          '@context': {
-            schema: 'http://schema.org/',
-            regen: 'https://schema.regen.network#',
-            qudt: 'http://qudt.org/schema/qudt/',
-            unit: 'http://qudt.org/vocab/unit/',
-            xsd: 'http://www.w3.org/2001/XMLSchema#',
-          },
```

Comment:
removing this part since this should be now fixed globally from jsonLdCompact directly